lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -583,6 +584,42 @@ void SessionStateEnforcer::m5g_process_response_from_upf(
   }
 }
 
+void SessionStateEnforcer::set_subscribed_qos(
+    SessionState& session_state, magma::SetSMSessionContextAccess* response) {

```

Comment:
```suggestion
    const SessionState& session_state, magma::SetSMSessionContextAccess* response) {
```
This should be const if it's not modified. If it's modified, let's make it a pointer

Reply:
Added `const` as suggested